homeassistant/components/weatherflow_cloud/sensor.py

```diff
@@ -223,6 +235,79 @@ class WeatherFlowCloudSensorEntityDescriptionWebsocketObservation(
         state_class=SensorStateClass.MEASUREMENT,
         suggested_display_precision=3,
     ),
+    # Rain Sensors
+    WeatherFlowCloudSensorEntityDescription(
+        key="precip_accum_last_1hr",
+        translation_key="precip_accum_last_1hr",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.precip_accum_last_1hr,
+        native_unit_of_measurement=UnitOfLength.MILLIMETERS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="precip_accum_local_day",
+        translation_key="precip_accum_local_day",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.precip_accum_local_day,
+        native_unit_of_measurement=UnitOfLength.MILLIMETERS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="precip_accum_local_day_final",
+        translation_key="precip_accum_local_day_final",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.precip_accum_local_day_final,
+        native_unit_of_measurement=UnitOfLength.MILLIMETERS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="precip_accum_local_yesterday",
+        translation_key="precip_accum_local_yesterday",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.precip_accum_local_yesterday,
+        native_unit_of_measurement=UnitOfLength.MILLIMETERS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="precip_accum_local_yesterday_final",
+        translation_key="precip_accum_local_yesterday_final",
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.precip_accum_local_yesterday_final,
+        native_unit_of_measurement=UnitOfLength.MILLIMETERS,
+    ),
+    WeatherFlowCloudSensorEntityDescription(
+        key="precip_analysis_type_yesterday",
+        translation_key="precip_analysis_type_yesterday",
+        device_class=SensorDeviceClass.ENUM,
+        options=["none", "rain", "snow", "sleet", "storm", "unknown"],
```

Comment:
unknown shouldn't be an option, we should remove it here and map it to `None` when we ever get it